src/methods/staking/payoutNominator.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Args,
+  BaseTxInfo,
+  createMethod,
+  UnsignedTransaction
+} from '../../util';
+
+export interface StakingPayoutNominatorArgs extends Args {
+  /**
+   * May not be lower than one following the most recently paid era. If it is
+   * higher, then it indicates an instruction to skip the payout of all
+   * previous eras.
+   */
+  era: number;
+  /**
+   * List of all validators that `who` had exposure to during `era`. If it is
```

Comment:
This is a tuple that also needs the weight on each validator. Should check with @thiolliere about how to calculate this. I wouldn't block merging just for this comment, but if we do then let's also make an issue for better docs on `payout_nominator`.

Reply:
ah in 
```
fn payout_nominator(origin, era: EraIndex, validators: Vec<(T::AccountId, u32)>)
```

validators is a tuple of validator account id and the index of the nominator in the exposure_clipped (i.e. in the `Staking::ErasStakersClipped::get(era, validator)`)

I'll update the doc

Reply:
documentation improved here https://github.com/paritytech/substrate/pull/5208